crates/re_space_view_spatial/src/scene_bounding_boxes.rs

```diff
@@ -51,10 +55,66 @@ impl SceneBoundingBoxes {
             self.current = self.current.union(*bbox);
         }
 
-        if self.accumulated.is_nothing() || !self.accumulated.size().is_finite() {
-            self.accumulated = self.current;
+        // Update smoothed bounding box.
+        let discontinuity = detect_boundingbox_discontinuity(self.current, previous);
+        if !self.smoothed.is_finite() || self.smoothed.is_nothing() || discontinuity {
+            // Reset the smoothed bounding box if it's not valid or we detect a discontinuity.
+            self.smoothed = self.current;
         } else {
-            self.accumulated = self.accumulated.union(self.current);
+            let dt = ui.input(|input| input.stable_dt.at_most(0.1));
+
+            // Smooth the bounding box by moving center & size towards the current bounding box.
+            let reach_this_factor = 0.9;
+            let in_this_many_seconds = 0.2;
+            let smoothing_factor =
+                egui::emath::exponential_smooth_factor(reach_this_factor, in_this_many_seconds, dt);
+
+            self.smoothed = macaw::BoundingBox::from_center_size(
+                self.smoothed
+                    .center()
+                    .lerp(self.current.center(), smoothing_factor),
+                self.smoothed
+                    .size()
+                    .lerp(self.current.size(), smoothing_factor),
+            );
+
+            if self.smoothed.min.distance_squared(self.current.min) > 0.001
+                || self.smoothed.max.distance_squared(self.current.max) > 0.001
```

Comment:
It would make more sense to compare this against something that is proportional to the size of the box